pom.xml

```diff
@@ -110,7 +110,7 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-			<version>2.5.4</version>
+			<version>2.7.9.4</version>
```

Comment:
There's a newer version than this, [`2.9.8`](https://search.maven.org/search?q=g:com.fasterxml.jackson.core%20AND%20a:jackson-databind)... any reason to pick this version specifically?

Reply:
And in fact... I can't find a version `2.7.9.4` in maven central... so I'm assuming this build _doesn't_ work.

Reply:
I used `2.7.9.4` because it was in the ticket https://jira.brightcove.com/browse/DELIVER-591
it builds fine locally but doesn't do on circle because this repo is on circle v1 which is no longer supported

Reply:
Cool, we can have a look tomorrow

Reply:
Weird, confirmed as well that 2.7.9.4 builds. I wonder why I can't find it on search.maven.org 🙃.

Definitely tomorrow-me's problem.

Reply:
We have the green ticket from Circle now and I've bumped jackson-databind to 2.9.8